src/components/ToastContainer.tsx

```diff
@@ -80,11 +82,11 @@ if (process.env.NODE_ENV !== 'production') {
     pauseOnHover: PropTypes.bool,
     closeOnClick: PropTypes.bool,
     newestOnTop: PropTypes.bool,
-    className: PropTypes.string,
+    className: PropTypes.any, //oneOfType([PropTypes.func, PropTypes.string]),
```

Comment:
I'll probably drop `prop-types` in a future release. So this is ok for me.

Reply:
Yes prop types are distressingly loose when we have good strict TS to work with